webui/main.js

```diff
@@ -64,6 +64,12 @@ function processWebuiSettingsJson (settings) {
 
   document.querySelector('#section-switcher').hidden = !settings.ShowNavigation
   document.querySelector('footer[title="footer"]').hidden = !settings.ShowFooter
+
+  if (settings.pageTitle) {
+    document.title = setttings.pageTitle;
```

Comment:
Looks like you've got a typo here on `setttings`, should be `settings`
```suggestion
    document.title = settings.pageTitle;
```